File: Python/Python_oppg_FVLA_Los/appendiks/oppg104.py
from numpy import *

def oppg104(a,b,m):
  ret = zeros(m)
  if m >= 1:
    ret[0] = b
  for k in range(1,m):
    ret[k] = a*ret[k-1]*(1-ret[k-1])
  return ret

File: Matlab/Matlab_oppg_FVLA_Los/appendiks/oppg104.m
function ret=oppg104(a,b,m)
  ret = zeros(1,m);
  if (m >= 1)
    ret(1) = b;
  end
  for (k=2:m)
    ret(k) = a*ret(k-1)*(1-ret(k-1));
  end
